src/common/Page.ts

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+import type { Readable } from 'stream';
```

Comment:
Does this impact agnostification? @jackfranklin

Reply:
I think this should be fine because the `import type` will not be included in the output of the ts compiler. But @jackfranklin  might know more about that :)

Reply:
Yup, `type` is OK :)

Reply:
Awesome, thanks! Do we have something in the try bots to check if puppeteer-common is indeed agnostic to the platform?